src/e2e/testUtils.ts

```diff
@@ -4,20 +4,18 @@
 
 const inputDelay = 120;
 
-const delay = (ms) =>
+export const delay = (ms: number) =>
   new Promise((resolve) => {
     setTimeout(resolve, ms);
   });
 
 /**
  * If there are messages, purge them.
- *
- * @returns {Promise} The promise from deleting messages
  */
-export const trashChat = () =>
-  game.messages.size > 0
-    ? game.messages.documentClass.deleteDocuments([], { deleteAll: true })
-    : null;
+export const trashChat = (): any => {
```

Comment:
Don't know if this passes linting for the current ESLint config. That said, we may have to revisit discussion of the FVTT Types project anyway...